addons/website_links/static/src/js/website_links_charts.js

```diff
@@ -240,7 +240,7 @@ publicWidget.registry.websiteLinksCharts = publicWidget.Widget.extend({
      * @private
      */
     _lastWeekClicksByCountry: function () {
-        var interval = moment().subtract(7, 'days').format('YYYY-MM-DD');
+        var interval = moment().subtract(7, 'days').locale('en').format('YYYY-MM-DD');
```

Comment:
It doesn't work with `local()` instead of specifying en ?
```suggestion
        var interval = moment().subtract(7, 'days').local().format('YYYY-MM-DD');
```